workflow-event-resource/src/out.py

```diff
@@ -15,11 +15,16 @@ def out(src_path, in_stream):
     id_path = data.pop("id")
 
     msg("Input: {}", input)
-    msg("Params: {}", data)
 
     with open(os.path.join(src_path, id_path), "r") as infile:
         id = infile.read()
 
+    if data.get("pdf_url"):
+        with open(os.path.join(src_path, "artifacts/pdf_url"), "r") as infile:
```

Comment:
Hardcoding `artifacts/pdf_url` in the path here causes some weirdness.
For example, if put `pdf_url: dslfajasjdfjdsljflkdsj`, I get a successful operation because pdf_url is never read.
```
Source dir /tmp/build/put

Input: {'source': {'api_root': 'https://cc1.cnx.org/api', 'status_id': 0}, 'params': {'pdf_url': 'dslfajasjdfjdsljflkdsj', 'status_id': '5'}}

Params: {'pdf_url': 'dslfajasjdfjdsljflkdsj', 'status_id': '5'}

Pdf_url: dslfajasjdfjdsljflkdsj

pdf_url: https://ce-pdf-spike.s3.amazonaws.com/artifacts/microbiology.pdf

++++++++++++++++

Updating status of event id 6

++++++++++++++++

src_path: /tmp/build/put

in_stream: <_io.TextIOWrapper name='<stdin>' mode='r' encoding='utf-8'>

api_root: https://cc1.cnx.org/api

id: 6

data: {'pdf_url': 'https://ce-pdf-spike.s3.amazonaws.com/artifacts/microbiology.pdf', 'status_id': '5'}

++++++++++++++++

response: {'collection_id': 'col12087', 'status_id': '5', 'content_server_id': '8', 'pdf_url': 'https://ce-pdf-spike.s3.amazonaws.com/artifacts/microbiology.pdf', 'version': '1.7', 'id': '6', 'created_at': '2019-11-20T19:45:27.153464', 'updated_at': '2019-11-20T20:23:51.362889', 'status': {'name': 'completed', 'id': '5'}, 'content_server': {'hostname': 'staging.cnx.org', 'host_url': 'https://staging.cnx.org', 'id': '8'}}

++++++++++++++++
```

Reply:
nice, catch on this one. but looking at this output, it looks like this resource hasn't been updated as expected.. those comments shouldn't be there.

Reply:
(Still editing):
- [ ] ran pipeline again to use updated resource (no display of extra messages)